WikiWatershed/model-my-watershed
# Update Mapshed bfi48grd and soilpallland2 rasters

## Overview

  The layers underlying the Recession Coefficient and SoilP estimates were made
  from geotiffs incorrectly converted from their original arcgis format -- this
  created layers with bad values. Update the layers to newly ingested,
  corrected layers

Connects #2293

### Demo

|               | Expected    | Approximate     |
|---------------|-------------|-----------------|
| soilpallland2 | 596.992     | 569.380416422   |
| bfi48grd      | 40.71185983 | 42.691618734836 |

Expected values show the output mean of the original rasters. Approximate values are created
by running the grouped average with a very simple CONUS shape. The CONUS shape included a little ocean which decreases the average a bit — you can see in the table though they're close enough we can assume correctness.

You can apply this patch to run the test. [national_test.patch.txt](https://github.com/WikiWatershed/model-my-watershed/files/1334897/national_test.patch.txt)

## Testing Instructions

 * Pull this branch and `vagrant ssh worker -c 'sudo service celeryd restart'`
 * Run some mapshed jobs and check the values of the exported GMS values per the instructions in #2263
